src/overlays/avg_parallelism_overlay.ts

```diff
@@ -282,13 +288,12 @@ export class AvgParallelismOverlay extends GenericSdfgOverlay {
         type: string,
         _ev: Event,
         _mousepos: Point2D,
-        _elements: SDFGElement[],
-        foreground_elem: SDFGElement,
+        _elements: Record<SDFGElementGroup, GraphElementInfo[]>,
```

Comment:
What is a record?

Reply:
A common supertype for dictionaries, (hash-)maps, etc. - in this case it's used to make a 'typed' dictionary where keys cannot be arbitrary strings, but of a specific set of strings (indicated by the type 'SDFGElementGroup')